externals/kyuubi-flink-sql-engine/src/main/scala/org/apache/kyuubi/engine/flink/operation/ExecuteStatement.scala

```diff
@@ -107,10 +105,13 @@ class ExecuteStatement(
       val operation = executor.parseStatement(sessionId, statement)
       operation match {
         case queryOperation: QueryOperation => runQueryOperation(queryOperation)
-        case setOperation: SetOperation => runSetOperation(setOperation)
-        case resetOperation: ResetOperation => runResetOperation(resetOperation)
+        case setOperation: SetOperation =>
```

Comment:
I mean can we support `SetOperation ` and `ResetOperation ` only in `PlanOnlyStatement `. And remove here.

Reply:
You have supported these two operations in `PlanOnlyStatement `, right? Maybe here, we can throw an exception then guide users to use `PlanOnlyStatement`, and make the `ExecuteStatement` operation, only support DML and DDL? I am not sure if we need to provide two entry points for this ability.

Reply:
@yanghua, in `SparkSQLOperationManager`, the `ExecuteStatement` and `PlanOnlyStatement` both support the `SetOperation` and `ResetOperation`. Refer to this, I perform the same implementation.

Reply:
You mean `SparkSQLOperationManager#newExecuteStatementOperation`？

> in SparkSQLOperationManager, the ExecuteStatement and PlanOnlyStatement both support the SetOperation and ResetOperation.

 where？

Reply:
@yanghua, I mean that in `SparkSQLOperationManager#newExecuteStatementOperation`, the `SetOperation` and `ResetOperation` are both supported in `PlanOnlyStatement` and `ExecuteStatement`. The behavior you mentioned above could be implemented in future.
cc @yaooqinn

Reply:
> @yanghua, I mean that in `SparkSQLOperationManager#newExecuteStatementOperation`, the `SetOperation` and `ResetOperation` are both supported in `PlanOnlyStatement` and `ExecuteStatement`. The behavior you mentioned above could be implemented in future. cc @yaooqinn

make sense to me